recipes/notices/top_banner_info/top_banner_info_variants.story.vue

```diff
@@ -12,7 +12,11 @@
         <span class="d-fw-bold"> Active Call </span> 3:21
       </template>
       <div class="d-d-flex d-c-pointer">
```

Comment:
Eck, should also be a `<button>`. It's using `d-c-pointer` just to get the cursor alone, but I don't see how events are attached to it.

Wrap it with a `button[type="button"]`, but a _plain_ one that doesn't add any new spacing or change to style. i suppose that might necessitate a combination of CSS utilities to override.

Reply:
And remove `d-c-pointer` once `<button>` is in there.

Reply:
And of course I find another non-button: https://github.com/dialpad/dialtone-vue/blob/afdd1467761fc16a4f850d9800e2478ade0125b8/recipes/notices/top_banner_info/top_banner_info_variants.story.vue#L100

"Click to answer"

Whatever combination of CSS utilities to not change its appearance. 🤦

Reply:
I mean this are just visual examples of what can you do with our components, they don't need events or something like that, but sure, I'll wrap them in a button to be more concise with other examples.